client/post-editor/editor-ground-control/test/index.jsx

```diff
@@ -60,218 +56,68 @@ describe( 'EditorGroundControl', function() {
 		} );
 	} );
 
-	beforeEach( function() {
-		ReactDom.unmountComponentAtNode( document.body );
-	} );
-
-	after( function() {
-		delete EditorGroundControl.prototype.__reactAutoBindMap.translate;
-		delete EditorGroundControl.prototype.__reactAutoBindMap.moment;
-		mockery.deregisterAll();
-		mockery.disable();
+	afterEach( function() {
+		sandbox.restore();
 	} );
 
 	describe( '#getPreviewLabel()', function() {
 		it( 'should return View if the site is a Jetpack site and the post is published', function() {
-			var tree = ReactDom.render(
+			const tree = TestUtils.renderIntoDocument(
```

Comment:
As background, the reason we'd opt to render to the `document.body` rather than use the `TestUtils` method is that the latter doesn't do any form of cleanup, so we'd frequently find tests to slow over time, or for warnings about maximum event listeners being exceeded. The [`renderIntoDocument` implementation](https://github.com/facebook/react/blob/622e84bcfe5630ca261f4652fe6a55f94b50103e/src/test/ReactTestUtils.js#L78-L86) is pretty naive. It's only been since recently (React 0.14?) that rendering to the body began to start logging a warning, hence the verbosity. In some of my recent single runner migrations, I've opted to try to replace these with [Enzyme `mount` or `shallow`](https://github.com/airbnb/enzyme) (e.g. #4550).